source/FilterableLogMonitor.js

```diff
@@ -103,19 +111,22 @@ export default class FilterableLogMonitor extends Component {
           fontSize: 14,
           minWidth: 200
         }}
+        onMouseOver={(e) => this.isMouseOver = true}
+        onMouseOut={(e) => this.isMouseOver = false}
```

Comment:
This approach doesn't feel quick right to me. I think instead, the logic should be this:
* If a user has not scrolled at all, it's okay to auto-scroll
* If a user scrolls up from the bottom, we should disable auto scroll
* If a user scrolls back down to the bottom, we can enable it again

Essentially the bottom can be a sticky area that the user can attach/detach from. I think using mouseover for this is _decent_ but could be annoying if people want to tab to another window to compare things and then when they scroll back, the position has changed again.

Reply:
fixed `onScroll` event